cvat/apps/auto_annotation/model_manager.py

```diff
@@ -256,27 +267,33 @@ def _create_polyshape(points, label, frame_number, attributes=None):
             "attributes": attributes or {},
         }
 
-def _process_detections(detections, path_to_conv_script):
+def _process_detections(detections, path_to_conv_script, run_as_admin=False):
     results = Results()
-    global_vars = {
-        "__builtins__": {
-            "str": str,
-            "int": int,
-            "float": float,
-            "max": max,
-            "min": min,
-            "range": range,
-            },
-        }
     local_vars = {
         "detections": detections,
         "results": results,
         }
-    exec (open(path_to_conv_script).read(), global_vars, local_vars)
+
+    if run_as_admin:
```

Comment:
Let's check here that the owner of the model is admin (has_admin_role). If it is true just run the model without any restrictions. What do you think?

Reply:
@nmanovic, do we want to add the owner instance to the method call of `_process_detections`? I think the `_run_inference_engine_annotation` method call would need to modified to include the primary key for the owner, but it's do-able.

Reply:
@benhoff, you are right.
- Let's rename `run_as_admin` by `restricted`.
- The flag should be True only if an owner of a model isn't admin.

Just let us run a model without restrictions if it was uploaded by an admin (trusted user which can do that even he/she wants). For regular users (owners) we will run models in the restricted environment.